README.md

```
@@ -250,6 +250,72 @@ generated from workouts are accepted without issues.
 }
 ```
 
+Using the `return_id` flag will make the code wait for Garmin to return
+its internal identifier for the newly created activity, which can
+be used for other methods such as renaming. The "internal ID" will
+be contained within the resulting dictionary at
+`result["detailedImportResult"]["successes"][0]["internalId"]`:
+
+> [!NOTE]  
+> Since this process waits for processing on the Garmin Connect server, be aware that
```

Markdownlint (0.37.0)

Expected: 80; Actual: 85
Line length

(MD013, line-length)